build.gradle

```diff
@@ -242,7 +244,27 @@ subprojects {
       }
     }
 
-    slf4j = { ->
+
+    slf4jApi = { ->
+      dependencies {
+        compileOnly group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+      }
+    }
+
+    slf4jLambdaApi = { ->
+      dependencies {
+        slf4jApi()
+        // It's so useful for us because we are using invocation of methods that potentially consume some time
+        // also in debug messages but this plugin allow us to using lambdas that generates log messages
+        // (mainly using string interpolation plugin) and values of this lambdas are calculated only when needed
+        // (when given log level is active)
```

Comment:
```suggestion
        // It's so useful for us because we are using invocations of methods that potentially consume some time
        // also in debug messages, but this plugin allows us to use lambdas that generate log messages
        // (mainly using string interpolation plugin) and values of this lambdas are calculated only when needed
        // (i.e. when the given log level is active)
```